lotus/models/lm.py

```diff
@@ -65,14 +66,19 @@ def __call__(self, messages: list[list[dict[str, str]]], **kwargs: dict[str, Any
             [self._get_top_choice_logprobs(resp) for resp in all_responses] if all_kwargs.get("logprobs") else None
         )
 
+        self.print_total_usage()
```

Comment:
Also there should probably be some `safe_mode` flag somewhere that determines whether this should be printed.